vnext/proposals/readme.md

```diff
@@ -13,7 +13,7 @@ The proposal process here takes its inspiration from the react-native-community
 
 ## Proposal process
 
-1. Follow the steps outlined in [Contribution guidelines](https://github.com/microsoft/react-native-windows/blob/master/vnext/docs/CONTRIBUTING.md) to set up your fork and branch to begin contributing.
+1. Follow the steps outlined in [Contribution guidelines](../../docs/contributing.md) to set up your fork and branch to begin contributing.
```

Comment:
>../.. [](start = 58, length = 5)

Does it work? Many web servers do not like relative paths like this.

Reply:
These files are in the repo so they aren't shown in the browser directly (you can view them in VS or VSCode though and it works fine there)